GetStream/stream-chat-android
# [3530] Improve slow mode

### 🎯 Goal

Closes #3530

Start countdown only after the message with attachments is sent.

### 🧪 Testing

1. Login as Jc or Dmitrii
2. Navigate to "Slow mode 10s" channel
3. Send a message with big attachments
4. Ensure that countdown begins only after the message was sent to the server

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
